R/deployApp.R

```diff
@@ -288,23 +292,26 @@ deployApp <- function(appDir = getwd(),
     python <- getPythonForTarget(python, accountDetails)
     pythonConfig <- pythonConfigurator(python, forceGeneratePythonEnvironment)
 
+    if (!quiet) {
+      cli::cli_alert_info("Bundling files: {.file {appFiles}}")
```

Comment:
cli will automatically truncate this list if it's long

Reply:
`bundleApp` can be (relatively) expensive. It would be nice to have similar to `withStatus` which marks the start/end of this type of operation (with timestamps). The existing `withStatus` is a little too awkward for this purpose, especially since `bundleApp` can produce output -- its "DONE" message expects it is emitted on the same line as the original status message.

Not for this PR, for sure; idle musing about better ways to mark the start and end of different processing phases.

Reply:
Oh I assumed that `bundleApp()` would be cheap. Is it just the cost of tar-ing up a bunch of files?

In general, would you be ok with moving to something that looks more like:

```
* Bundling files...
✅ Bundling complete
```

The dangling line makes all sorts of output (including errors) look wonky.

Reply:
`bundleApp` includes manifest creation, which means dependency discovery (finding explicit package references, then exploding the entire package requirement tree). It has been expensive in the past. https://github.com/rstudio/packrat/pull/615, for example.